pyrobot/plugins/__init__.py

```diff
@@ -0,0 +1,19 @@
+from pyrobot import logger
+
+
+def __list_all_modules():
+    from os.path import dirname, basename, isfile
+    import glob
+
+    mod_paths = glob.glob(dirname(__file__) + "/*.py")
+    all_modules = [
+        basename(f)[:-3]
+        for f in mod_paths
+        if isfile(f) and f.endswith(".py") and not f.endswith("__init__.py")
+    ]
+    return all_modules
+
+
+ALL_MODULES = sorted(__list_all_modules())
+logger.info("Modules to load:\n{}\n".format(", ".join(ALL_MODULES)))
+__all__ = ALL_MODULES + ["ALL_MODULES"]
```

Comment:
we don't need the @banHammerMarie_bot way of imports. We are using https://docs.pyrogram.org/topics/smart-plugins which is documented officially.